ratis-docs/themes/ratisdoc/static/js/jquery-3.5.1.min.js

```diff
@@ -1,2 +1,2 @@
-/*! jQuery v3.4.1 | (c) JS Foundation and other contributors | jquery.org/license */
+/*! jQuery v3.5.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
There doesn't seem to be any change apart from jquery version number in the jquery-3.5.1.min.js file?